javers-core/src/main/java/org/javers/core/metamodel/property/BeanBasedPropertyScanner.java

```diff
@@ -31,4 +31,12 @@ public BeanBasedPropertyScanner(AnnotationNamesProvider annotationNamesProvider)
         }
         return beanProperties;
     }
+
+    @Override
+    public Property scanSingleProperty(Class<?> managedClass, String propertyName) {
+        JaversMethod persistentGetter = ReflectionUtil.getPersistentGetter(managedClass, propertyName);
+
+
+        return null;
```

Comment:
Testy są tylko na javersie zbudowanym z FieldPropertyScannerem :/ chyba powinienem dodać jeszcze jedną zmienną do tabelki where: . Co do filtrowania to był mój pierwszy pomysł żeby tak zrobić jednak odszedłem od niego ze względu na to, że w java api są wystawione metody do pobierania pojedyńczych fieldów i jakoś tak tym się pokierowałem, ale może rzeczywiście w tym przypadku warto to po prostu odfiltrować "u klienta"... sam nie wiem